packages/graphile-build-pg/src/plugins/PgIntrospectionPlugin.d.ts

```diff
@@ -1,3 +1,5 @@
+import { SchemaBuilder, Options } from "../../../graphile-build/src";
```

Comment:
We cannot do this relative require into a different package. Instead:

```suggestion
import { SchemaBuilder, Options } from "graphile-build";
```

Reply:
(Apparently I cannot push this fix myself, so I'll wait on you to accept it. @ me when it's ready.)

Reply:
I see, thanks. @benjie I just committed it.